### PR TITLE
Be able to load maps with custom pool size and max waiting clients

### PR DESCRIPTION
**Pull request by dgaubert**

### CODE REVIEW

File: lib/mapnik_backend.js [hunk 1]
```diff
@@ -220,51 +227,91 @@ MapnikSource.prototype._createPool = function(xml) {
...
+                map = null;
+
+                return resolve();
+            });
+        }
     };
 
-    this._pool = Pool(factory);
+    const options = {
+        max: this._uri.query.poolSize || cpusNumber,
```
Comment (simon-contreras-deel) [inline lib/mapnik_backend.js:265]: Why don't we use `UV_THREADPOOL_SIZE`?

File: lib/mapnik_backend.js [hunk 2]
```diff
@@ -220,51 +227,91 @@ MapnikSource.prototype._createPool = function(xml) {
...
+        maxWaitingClients: this._uri.query.poolMaxWaitingClients ? this._uri.query.poolMaxWaitingClients : 32
```
Comment (simon-contreras-deel) [inline lib/mapnik_backend.js:266]: I think the default value should be a multiplier of `max`

**Review by simon-contreras-deel — APPROVED** [review]
Some comments, but it LGTM

File: lib/mapnik_backend.js (hunk 1 above)
Comment (dgaubert) [reply lib/mapnik_backend.js:265]: Currently, we set `UV_THREADPOOL_SIZE` to `96` or `128` typically. These values are too high for a normal use of map pool renderers.

File: lib/mapnik_backend.js (hunk 2 above)
Comment (dgaubert) [reply lib/mapnik_backend.js:266]: Let's see how these values work first.

**Pull request merged**